src/broker/overlay.c

```diff
@@ -593,6 +564,54 @@ void overlay_set_monitor_cb (struct overlay *ov,
     ov->child_monitor_arg = arg;
 }
 
+static json_t *list_object_create (struct overlay *ov)
```

Comment:
whenever I see a function starting with `list_` I think of a singly linked list.  Perhaps rename function?